README.md

```diff
@@ -4,9 +4,7 @@ A Crystal library for creating and running tasks
 
 ## Installing the CLI by Homebrew
 
-1.  Install [homebrew](http://brew.sh)
-2.  Run `brew tap luckyframework/lucky`
-3.  Run `brew install lucky`
+To install the Lucky CLI, read the guides for your Operating System https://luckyframework.org/guides/getting-started/installing
```

Comment:
This will need to replace the building from source that's below. I'm thinking both of the sections go away, and it's replaced by a single block that says "Installing Lucky". Then the URL should be a clickable link.

Then the next section below would saying something along the lines of "Development" with a sub sections of "building the CLI. That subsection could then detail out the steps to build from source.

Reply:
hey @jwoertink ,I got your point ,I'll make the required changes according to your suggestions.

Reply:
hey @jwoertink I have done changes according to your suggestions can u please check it . 
can u review this once https://github.com/luckyframework/lucky_cli/pull/683/commits/e6bcc50b5f0acb7c09907370abdd2928c66618ca